<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="3" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Login Page loaded successfully..]]>
    </line>
    <line>
      <![CDATA[Driver instance closed..]]>
    </line>
    <line>
      <![CDATA[Login Page loaded successfully..]]>
    </line>
    <line>
      <![CDATA[Driver instance closed..]]>
    </line>
    <line>
      <![CDATA[Login Page loaded successfully..]]>
    </line>
    <line>
      <![CDATA[Driver instance closed..]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="32234" started-at="2019-03-04T18:24:04Z" finished-at="2019-03-04T18:24:36Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="32234" started-at="2019-03-04T18:24:04Z" finished-at="2019-03-04T18:24:36Z">
      <class name="com.qa.testcases.LoginPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.testcases.LoginPageTest@76a3e297]" name="setUp" is-config="true" duration-ms="10759" started-at="2019-03-04T18:24:04Z" finished-at="2019-03-04T18:24:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="loginTest(java.util.Map)[pri:1, instance:com.qa.testcases.LoginPageTest@76a3e297]" name="loginTest" duration-ms="617" started-at="2019-03-04T18:24:15Z" description="QT-2543 Valid Login" data-provider="getLoginTestData" finished-at="2019-03-04T18:24:16Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Role=System Admin, Username=testing@deskpro.com, Password=HGdBBRjikAmlCotNQvzecAvXJotfWCnM}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.testcases.LoginPageTest.loginTest(LoginPageTest.java:60)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[Login Page loaded successfully..]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.testcases.LoginPageTest@76a3e297]" name="tearDown" is-config="true" duration-ms="104" started-at="2019-03-04T18:24:16Z" finished-at="2019-03-04T18:24:16Z">
          <reporter-output>
            <line>
              <![CDATA[Driver instance closed..]]>
            </line>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.testcases.LoginPageTest@76a3e297]" name="setUp" is-config="true" duration-ms="8938" started-at="2019-03-04T18:24:16Z" finished-at="2019-03-04T18:24:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="loginTest(java.util.Map)[pri:1, instance:com.qa.testcases.LoginPageTest@76a3e297]" name="loginTest" duration-ms="499" started-at="2019-03-04T18:24:25Z" description="QT-2543 Valid Login" data-provider="getLoginTestData" finished-at="2019-03-04T18:24:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Role=User Admins, Username=testing@deskpro.com, Password=HGdBBRjikAmlCotNQvzecAvXJotfWCnM}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.testcases.LoginPageTest.loginTest(LoginPageTest.java:60)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[Login Page loaded successfully..]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.testcases.LoginPageTest@76a3e297]" name="tearDown" is-config="true" duration-ms="92" started-at="2019-03-04T18:24:26Z" finished-at="2019-03-04T18:24:26Z">
          <reporter-output>
            <line>
              <![CDATA[Driver instance closed..]]>
            </line>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.qa.testcases.LoginPageTest@76a3e297]" name="setUp" is-config="true" duration-ms="8685" started-at="2019-03-04T18:24:26Z" finished-at="2019-03-04T18:24:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="loginTest(java.util.Map)[pri:1, instance:com.qa.testcases.LoginPageTest@76a3e297]" name="loginTest" duration-ms="533" started-at="2019-03-04T18:24:35Z" description="QT-2543 Valid Login" data-provider="getLoginTestData" finished-at="2019-03-04T18:24:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[{Role=System Admin, Username=testing@deskpro.com, Password=HGdBBRjikAmlCotNQvzecAvXJotfWCnM}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at com.qa.testcases.LoginPageTest.loginTest(LoginPageTest.java:60)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
            <line>
              <![CDATA[Login Page loaded successfully..]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.qa.testcases.LoginPageTest@76a3e297]" name="tearDown" is-config="true" duration-ms="112" started-at="2019-03-04T18:24:36Z" finished-at="2019-03-04T18:24:36Z">
          <reporter-output>
            <line>
              <![CDATA[Driver instance closed..]]>
            </line>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.qa.testcases.LoginPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
